### PR TITLE
Remove pending from phone number editing specs

### PR DESCRIPTION
**Pull request by saraOSI**
[#refactor]

### Pivotal Story

- [Redux phone number refactor #152463644](https://www.pivotaltracker.com/story/show/152463644)

### Purpose
- Fix participant phone number specs

### CODE REVIEW

File: spec/features/screening/participant/phone_number_spec.rb [hunk 1]
```diff
@@ -57,15 +53,17 @@
     within edit_participant_card_selector(marge.id) do
       expect(page).to have_field('Phone Number', with: '(917)555-5555')
       expect(page).to have_field('Phone Number Type', with: 'Work')
-      fill_in 'Phone Number', with: '789-456-1245'
+      fill_in 'Phone Number', with: '7894561245'
```
Comment (jgedeon-osi) [inline spec/features/screening/participant/phone_number_spec.rb]: Why did you remove the dashes?

Comment (saraOSI) [reply spec/features/screening/participant/phone_number_spec.rb]: We have a mask on the phone number, so the dashes are eaten up by the input. They aren't allowed characters. I guess this might be implicitly testing that? If we want to test that, though, I think making it more clear would be wise.

Comment (jgedeon-osi) [reply spec/features/screening/participant/phone_number_spec.rb]: Yeah or just ensuring there is a test for non numeric inputs.
